src/Documentation/sidebar.json

```diff
@@ -162,7 +163,8 @@
       "unprotect.md": "unprotect",
       "cache.md": "cache",
       "cache:dir.md": "cache dir",
-      "commit.md": "commit"
+	  "commit.md": "commit",
```

Comment:
please fix indent

Reply:
![Screenshot from 2019-05-11 03-35-46](https://user-images.githubusercontent.com/35191225/57559106-eaed7100-739d-11e9-8996-25b4496ca2a2.png)
No indent is being shown in my codebase.

Reply:
Tabs vs spaces?

Reply:
@shcheklein I used tabs for indentation.